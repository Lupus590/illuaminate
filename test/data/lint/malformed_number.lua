-- config: (lint (only (syntax:malformed-number)))
local _ = {
  2..,
  2...,
  2.2..,
  2e2.2,

  -- ok
  2.3,
  2 .. 0,
  2. .. 0,
  2e3 .. 0,
  0x2 ..0,
}
