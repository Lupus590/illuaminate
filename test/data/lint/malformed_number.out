malformed_number.lua:[3:3-3:5]: Malformed number [syntax:malformed-number]
   │
 3 │   2..,
   │   ^^^
malformed_number.lua:[4:3-4:6]: Malformed number [syntax:malformed-number]
   │
 4 │   2...,
   │   ^^^^
malformed_number.lua:[5:3-5:7]: Malformed number [syntax:malformed-number]
   │
 5 │   2.2..,
   │   ^^^^^
malformed_number.lua:[6:3-6:7]: Malformed number [syntax:malformed-number]
   │
 6 │   2e2.2,
   │   ^^^^^
4 errors and 0 warnings
